.github/workflows/release.yml

```diff
@@ -45,6 +45,8 @@ jobs:
             macOS ${{matrix.macos-vsn}} (64 bit).
           fail_on_unmatched_files: true
           # yamllint disable-line rule:line-length
-          files: ${{runner.temp}}/otp/macos64-${{matrix.macos-vsn}}-OTP-${{steps.configure.outputs.otp_vsn}}.tar.gz
+          files: |
+            ${{runner.temp}}/otp/macos64-${{matrix.macos-vsn}}-OTP-${{steps.configure.outputs.otp_vsn}}.tar.gz
```

Comment:
The naming conventions here are odd, but we can back to it, I wanted to get this in now, but not break convention without discussion.